conmon/conmon.c

```diff
@@ -1259,10 +1261,17 @@ int main(int argc, char *argv[])
 			 NULL);
 
 	if (opt_exec) {
-		add_argv(runtime_argv,
-			 "exec", "-d",
+		if (opt_detach) {
+			add_argv(runtime_argv,
+			 "exec", "-d"
```

Comment:
I'd go with --detach instead of -d here.  It's just clearer code maintenance wise and protects from someone later on from grabbing -d for something else; granted an unlikely situation.

Reply:
Ok that was existing code, but I see your point.